### PR TITLE
Update Readme for 7.1.0

### PR DESCRIPTION
**Pull request by anukalp**
Version updates

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,17 +31,17 @@ To download the latest SDK as a JAR, please visit [http://inmobi.com/sdk](https:
 **To continue integrating with the InMobi SDK, please see the [Integration Guidelines](https://support.inmobi.com/monetize/android-guidelines/) for Android.**
 
 ## New in this version
-- InMobi SDK v7.0.0 for Android introduces brand new Native Ad solution.
+- InMobi SDK v7.1.0 for Android introduces brand new Native Ad solution.
```
Comment (SwagatoMondal) [inline README.md]: Remove this change.

File: README.md [hunk 2]
```diff
@@ -31,17 +31,17 @@ To download the latest SDK as a JAR, please visit [http://inmobi.com/sdk](https:
...
 - Interactive Video Ad Experience
 - Engaging end-cards are now supported via Rich-Media End-cards
 - Support for GIF images
 
 ## Requirements
 - Android 4.0.1 (API level 15) and higher
-- android-support-v4.jar, r24 
-- android-recyclerview-v7.jar,r24
+- android-support-v4.jar, r27.1.0
```
Comment (SwagatoMondal) [inline README.md]: Add only r27

**Review by SwagatoMondal — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@ InMobi SDK for Android
 
 Modified: Sep 18, 2017
```
Comment (SwagatoMondal) [inline README.md]: Date has to be changed

**Review by SwagatoMondal — CHANGES_REQUESTED** [review]

**Review by SwagatoMondal — APPROVED** [review]

**Review by kvaddamani — APPROVED** [review]
LGTM

**Pull request merged**